src/Microsoft.Health.SqlServer/Configs/SqlClientRetryOptions.cs

```diff
@@ -41,7 +41,7 @@ public class SqlClientRetryOptions
                 {
                     // Default .NET errors:
                     1204,   // The instance of the SQL Server Database Engine cannot obtain a LOCK resource at this time. Rerun your statement when there are fewer active users. Ask the database administrator to check the lock and memory configuration for this instance, or to check for long-running transactions.
-                    1205,   // Transaction (Process ID) was deadlocked on resources with another process and has been chosen as the deadlock victim. Rerun the transaction
+                    1205,   // Transaction (Process ID) was deadlocked on resources with another process and has been chosen as the deadlock victim. Rerun the transaction.
```

Comment:
>. [](http://example.com/codeflow?start=170&length=1)

Not all rows have period at the end anyway...

Reply:
I'm not adding periods to the rows ending with sentences between parenthesis, just to the others.